src/scripts/main.js

```diff
@@ -1 +1,68 @@
 'use strict';
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  })
```

Comment:
```suggestion
  });
```
don't forget about semicolon. fix everywhere